tests/tests.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+# Streisand syntax check.
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+export ANSIBLE_CONFIG=$DIR/ansible.cfg
+
+set -e
+
+command -v ansible > /dev/null 2>&1 || {
+  echo "Ansible is not installed."
+  exit 1
+}
+
+required_ansible_version="2.3.0.0"
+
+if [[ "$(ansible --version | grep -oe '2\(.[0-9]\)*')" < $required_ansible_version ]]
+then
+    echo "Ansible $required_ansible_version or higher is required."
+    exit 1
+fi
```

Comment:
I was thinking something simpler like putting the common lines in a .sh file that both the striesand script and the test script source directly.